banzai/qc/header_checker.py

```diff
@@ -0,0 +1,155 @@
+"""
+This module performs controls on several critical parameters of image header, contains in
+header_expected_format dictionnary.
+@author:ebachelet
+"""
+from banzai.stages import Stage
+from banzai import logs
+
+class HeaderSanity(Stage):
+    """
+       This class contains the needed function/definitions to performs the header check.
+    """
+
+    def __init__(self, pipeline_context):
+        super(HeaderSanity, self).__init__(pipeline_context)
+
+        self.header_expected_format = {'RA': str, 'DEC': str, 'CAT-RA': str, 'CAT-DEC': str,
+                                       'OFST-RA': str, 'OFST-DEC': str, 'TPT-RA': str,
+                                       'TPT-DEC': str, 'PM-RA': str, 'PM-DEC': str,
+                                       'CRVAL1': float, 'CRVAL2': float, 'CRPIX1': int,
+                                       'CRPIX2': int, 'EXPTIME': float}
+
+    @property
+    def group_by_keywords(self):
+        return None
+
+    def do_stage(self, images):
+        """ Performs several checks on the header, see the relative functions.
+        :param list images: a list of image object.
+        :returns: the list of image object after check
```

Comment:
I think we decided a while ago to not reject anything from this module, because some info could be found again easily, examples date of observation or RA DEC